### PR TITLE
QF-1760 Store or retain a value for standard version date version approved

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Data/OrganisationStandardRepository.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public async Task<OrganisationStandardVersion> UpdateOrganisationStandardVersion
                         SET
                             [EffectiveFrom] = @effectiveFrom,
                             [EffectiveTo] = @effectiveTo,
-                            [DateVersionApproved] = @dateVersionApproved,
+                            [DateVersionApproved] = COALESCE(DateVersionApproved, @dateVersionApproved),
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.AssessorService.Data/OrganisationStandardRepository.cs:71]: @ChrisJWoodcock is DateVersionApproved used anywhere at all, if not could this be removed?

File: src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs [hunk 2]
```diff
@@ -65,7 +65,7 @@ public async Task<OrganisationStandardVersion> Handle(OrganisationStandardVersio
                     OrganisationStandardId = organisationStandard.Id,
                     EffectiveFrom = request.EffectiveFrom,
                     EffectiveTo = request.EffectiveTo,
-                    DateVersionApproved = null,
+                    DateVersionApproved = DateTime.UtcNow,
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs:68]: @ChrisJWoodcock have you updated the related unit tests?

File: src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptOutHandler.cs [hunk 3]
```diff
@@ -59,7 +59,7 @@ public async Task<OrganisationStandardVersion> Handle(OrganisationStandardVersio
                     OrganisationStandardId = organisationStandard.Id,
                     EffectiveFrom = request.EffectiveFrom,
                     EffectiveTo = request.EffectiveTo,
-                    DateVersionApproved = null,
+                    DateVersionApproved = DateTime.UtcNow,
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptOutHandler.cs:62]: @ChrisJWoodcock have you updated related unit test?

File: src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs]: Good spot - I had not... I have now and made some changes to make it properly testable

**Review by Jordan-fujitsu — APPROVED** [review]

**Pull request merged**